tests/JustSaying.IntegrationTests/Fluent/Configuration/Approvals/WhenUsingNamingConventions.ThenTheNamingConventionIsApplied.approved.txt

```diff
@@ -0,0 +1,55 @@
+{
+  "Region": "eu-west-1",
+  "Middleware": {
+    "Middlewares": [
+      {
+        "MessageType": "SimpleMessage",
+        "QueueName": "beforequeue-simplemessage-afterqueue",
+        "MiddlewareChain": [
+          "MessageContextAccessorMiddleware",
+          "ErrorHandlerMiddleware",
+          "LoggingMiddleware",
+          "StopwatchMiddleware",
+          "SqsPostProcessorMiddleware",
+          "HandlerInvocationMiddleware`1"
```

Comment:
I wonder if these should print out including the type argument's type name?

Reply:
Yeah that's a good shout, let me know what you think about the [latest commit](https://github.com/justeat/JustSaying/pull/973/commits/a5f9cb64f7c0e2ef092484d2920c5ce84e874716) 👍